components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -451,6 +454,18 @@ public abstract AuthenticationStep[] getConfiguredAuthenticators(String applicat
     public abstract AuthenticationStep[] getConfiguredAuthenticators(String applicationID)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Get the list of trusted mobile applications based on the requested platform type.
```

Comment:
```suggestion
     * Get the list of trusted applications based on the requested platform type.
```

Reply:
Removing the term mobile as even though we naturally expect a mobile application here we don't have any 1st class support for "mobile" apps. And from the perspective of IS they are all app configs

Reply:
Lets update this in other places as well. I can see few other comments referring as mobile applications.